modules/ossm-exposing-a-service-by-using-the-kubernetes-gateway-api.adoc

```diff
@@ -61,40 +80,38 @@ spec:
 $ oc apply -f httpbin-gw.yaml
 ----
 
-. Create a YAML file named `httpbin-hr.yaml` that defines an `HTTPRoute` resource. The `HTTPRoute` resource specifies the rules that route traffic from the gateway proxy to the `httpbin` service.
+. Create a YAML file named `httpbin-ingress-hr.yaml` that defines an `HTTPRoute` resource for the ingress gateway, similar to the following example:
 +
-.Example HTTPRoute file
-[source,yaml,subs="attributes,verbatim"]
+[source,yaml]
 ----
 apiVersion: gateway.networking.k8s.io/v1
 kind: HTTPRoute
 metadata:
-  name: httpbin
+  name: httpbin-ingress
   namespace: httpbin
 spec:
-  parentRefs: # <1>
+  parentRefs:
   - name: httpbin-gateway
     namespace: httpbin
+  hostnames:
+  - "httpbin.example.com"
   rules:
-  - matches:
-    - path:
-        type: PathPrefix
-        value: /status
-    - path:
-        type: PathPrefix
-        value: /headers
-    backendRefs: # <2>
+  - backendRefs:
```

Comment:
The matches should not be removed, otherwise the validation below (i.e., request to `/get`) will fail in the subsequent steps below. Please add the rules.matches.path to the YAML file.